feat: Add new Badges (#1392)
cozy/cozy-ui: react/Badge/Readme.md

`<Badge />` component wraps the element on which you want to apply a badge. In the following examples, it is applied to compact `<ButtonAction />`.

> You should use `<Badge />` when we want to add additionnal informations on the wrapped component

Reviewer: ```suggestion
You should use `<Badge />` to provide additional information related to the wrapped content.
```

Just a suggestion, both are correct but I don't like the you/we switch :)

  </p>
</div>
```

#### Alignment

> Only support `bottom-right` 

Reviewer: ```suggestion
Only supports `bottom-right` 
```

```
import Badge from 'cozy-ui/transpiled/react/Badge'
import Icon from 'cozy-ui/transpiled/react/Icon';

<div>
  <p>
    <Badge 
      content={ <Icon icon="link" size={10} />} type="normal" alignment="bottom-right" size='medium'>

>       <Icon icon="cloud" size="32" />

Reviewer: Could we use a `ButtonAction ` here as well? I find it's a bit easier to understand which one the badge is and which one the other component is if we don't use Icon for both :)

Author: It was to have a really life example since we currently don't use Badge with a ButtonAction

Reviewer: Mmh ok fair enough!

    </Badge>
  </p>
  
</div>

> ```

Reviewer 2: I am not sure why we add doc for `alignment` since it can only have one value ?

Author: For now, but we plan to add top-right pretty soon
